examples/hacker_news_pwa/web/index.html

```diff
@@ -59,7 +59,7 @@
     nav a:hover {
       color: #fff;
     }
-    nav a.active {
+    nav a:active {
```

Comment:
Did you test this? This wasn't a typo. This is a specific class added by the routerLink directive, that the router adds/removes when the link is active.

https://github.com/dart-lang/angular/blob/843cb0fb5972b6a335a07ca65fd54273fe08a328/examples/hacker_news_pwa/lib/app_component.html#L6